server/chat-commands/core.ts

```diff
@@ -796,7 +811,8 @@ exports.commands = {
 				if (playerUser.id === user.id) {
 					battle.allowExtraction[user.id].add(user.id);
 				} else {
-					playerUser.sendTo(room, Chat.html`|html|${user.name} wants to extract the battle input log. <button name="send" value="/allowexportinputlog ${user.id}">Share your team and choices with "${user.name}"</button>`);
+					const msg = `${user.name} wants to extract the battle input log. <button name="send" value="/allowexportinputlog ${user.id}">Share your team and choices with "${user.name}"</button>`;
+					playerUser.sendTo(room, Chat.html`|html|${msg}`);
```

Comment:
This will completely fail to render the button. Try it out and you'll see what's wrong.

In the future, just do the multiline thing like I suggested for `allowexportinputlog`.